oidc-service-impl/src/test/java/io/mosip/esignet/services/ConsentHelperServiceTest.java

```diff
@@ -0,0 +1,220 @@
+package io.mosip.esignet.services;
+
+import io.mosip.esignet.api.dto.ClaimDetail;
+import io.mosip.esignet.api.dto.Claims;
+import io.mosip.esignet.core.dto.*;
+import io.mosip.esignet.core.spi.ConsentService;
+import io.mosip.esignet.core.util.KafkaHelperService;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@RunWith(MockitoJUnitRunner.class)
+@Slf4j
+public class ConsentHelperServiceTest {
+
+
+
+    @Mock
+    CacheUtilService cacheUtilService;
+
+    @Mock
+    ConsentService consentService;
+
+    @Mock
+    KafkaHelperService kafkaHelperService;
+
+    @InjectMocks
+    ConsentHelperService consentHelperService;
+
+
+    @Test
+    public void addUserConsent_withValidDetails_thenPass()
```

Comment:
Lets add one more case for consentHelperService.addUserConsent( with linked as false

Reply:
Done

Reply:
Have a look